autoload/go/alternate.vim

```diff
@@ -0,0 +1,36 @@
+" By default use edit to switch to the new alternate buffer
+if !exists("g:go_alternate_mode")
+	let g:go_alternate_mode = "edit"
+endif
+
+" By default create the test file if it does not exist
+if !exists("g:go_create_test_file")
+	let g:go_create_test_file  = 1
```

Comment:
I think we should change that. The bang should create a file if passed if not it should give an error: `no test file found. Call :GoAlternate! to create one`. With that also remove the   `let g:go_create_test_file = 1` option completely.